src/components/Widgets/Markdown/MarkdownControl/VisualEditor/__tests__/__snapshots__/parser.spec.js.snap

```diff
@@ -10,7 +10,11 @@ Object {
         Object {
           "data": undefined,
           "kind": "text",
-          "text": "H1",
+          "ranges": Array [
```

Comment:
I didn't see in the changelog exactly why this was needed, could you explain?

Reply:
It's actually not in the changelog, but Slate text nodes used to come in two flavors, either they'd have a simple "text" property with the value, or else a "ranges" property having an array of objects representing text with associated marks. They quietly switched to using "ranges" only.